example/lambdas/python-reference-task/package.json

```diff
@@ -21,7 +21,8 @@
   },
   "ava": {
     "serial": true,
-    "timeout": "15m"
+    "timeout": "15m",
+    "failFast": true
```

Comment:
Is this intended to be a permanent change?      Having smaller packages fail fast may be a detriment in situations where devs are using CI to offload full test runs.        Not an objection, just looking for clarification on intent.

Reply:
It _was_ intended to be a permanent solution. The downside (as you say) is a situation where you run the full suite, see a failure, fix it, and then something else fails. Slower overall process than seeing all the failures at once.

The upshot is that you get your results faster. How much faster is debatable because this is per-package but my thinking was you'd generally rather stop at the first failure because you're roughly aware of which units might break based on your changes. 

Open to push back here. We had the failFast (previously `fail-fast`) flag set on a few packages already (api, async-ops, etc.). If we decide not to do it I think it makes sense to remove it from all packages. Having a mixed bag might not make a ton of sense?